src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Dashboard from './components/Dashboard';
+
+const App: React.FC = () => {
+  return (
+    <div className="App">
```

Comment:
Please consider naming classNames in lowercase as well

Reply:
done